tasks/views.py

```diff
@@ -1,3 +1,35 @@
 from django.shortcuts import render
 
 # Create your views here.
+from rest_framework import permissions
+from rest_framework.viewsets import ModelViewSet
+
+from tasks.models import Task
+from tasks.serializers import TaskListSerializer, TaskSerializer
+
+
+class IsStaffOrReadOnly(permissions.BasePermission):
```

Comment:
Seems like this is a clone of the same class in announcements/views.py? If so then maybe we should split them into a single file to avoid code duplication

Reply:
Well, honestly I thought about it. But I also want to have something separated that I can just simply copy this app and put it into another future project, without adding any codes. That's why I didn't merged it into one file.
Anyway, I will change it (maybe)